### ISSUE TITLE
Better focus management

### ISSUE THREAD

**Issue by markus-walther** [axa-ch-webhub-cloud/inno-day-time-boii#24]
When navigating to a new day, users doing batch data entry expect focus to be on the first unfilled input field.

Implement this.

Debatable: Tab order currently includes delete action buttons. Arguably they are rare. Maybe they should be skipped in tabbing?

**Comment by danyball** [comment]
There is no input field in which we can set the focus. And focussing the + could be a problem if you switch between days.

**Comment by markus-walther** [comment]
This is **not** fixed correctly. The description said first _unfilled_ field. This does not happen in all cases.
